src/Analyzers/Core/CodeFixes/ImplementInterface/AbstractImplementInterfaceService.cs

```diff
@@ -94,4 +98,58 @@ public async Task<Document> ImplementInterfaceAsync(
             this, document, info, options, configuration);
         return await generator.ImplementInterfaceAsync(cancellationToken).ConfigureAwait(false);
     }
+
+    public async Task<ISymbol> ExplicitlyImplementSingleInterfaceMemberAsync(
+        Document document,
+        IImplementInterfaceInfo info,
+        ISymbol member,
+        ImplementTypeOptions options,
+        CancellationToken cancellationToken)
+    {
+        var configuration = new ImplementInterfaceConfiguration()
+        {
+            Abstractly = false,
+            Explicitly = true,
+            OnlyRemaining = false,
+            ImplementDisposePattern = false,
+            ThroughMember = null,
+        };
+        var generator = new ImplementInterfaceGenerator(
+            this, document, info, options, configuration);
+        var implementedMembers = await generator.GenerateExplicitlyImplementedMembersAsync(member, options.PropertyGenerationBehavior, cancellationToken).ConfigureAwait(false);
+        cancellationToken.ThrowIfCancellationRequested();
+
+        var singleImplemented = implementedMembers[0]!;
```

Comment:
why the !

Reply:
Removing the ! and asserting not null as a contract

Reply:
Removed